src/js/functions/getFalseData.js

```diff
@@ -0,0 +1,14 @@
+import falseNegativeData from '../../../data/exploration/wrong-classifications/complementNB/false_negatives1.json';
+import falsePositiveData from '../../../data/exploration/wrong-classifications/complementNB/false_positives1.json';
+
+const getFalseData = () => {
+  return {
+    falseNegaties: Object.keys(falseNegativeData.actual_label).length,
+    falsePositives: Object.keys(falsePositiveData.actual_label).length
```

Comment:
One thing worth noting is that this dataset has `prediction_label` as a subset of data. I wonder if those can be used for the stacked bar example.

Reply:
good thought, but this is the same data as the actual_label. This is just what determines that a particular tweet within the the train was incorrectly labeled based off our model. I'm going to have source the actual counts from the training data set.

Reply:
good thought, but I'll actually need to source this info from somewhere else. The prediction_label is really the same as the actual_label, it's really just used to determine the training data that was incorrectly predicted based on the actual results.